packages/rule-minified-js/README.md

```diff
@@ -0,0 +1,106 @@
+# minified-js (`@sonarwhal/rule-minified-js`)
```

Comment:
Change `minified-js` for something like `Validate if the JavaScript is minified`